### ISSUE TITLE
ResponsiveWrapper is not working 

### ISSUE THREAD

**Issue by NumanAhmad1** [Codelessly/ResponsiveFramework#167]
when i use ResponsiveWrapper it gives an error that it is an undefined class, i have tried it with .builder but still not working, but when i use ResponsiveBreakpoints then it works.

**Comment by abigotado** [comment]
There is no ResponsiveWrapper in the latest version. See the migration guide:

[https://github.com/Codelessly/ResponsiveFramework/blob/master/migration_0.2.0_to_1.0.0.md](https://github.com/Codelessly/ResponsiveFramework/blob/master/migration_0.2.0_to_1.0.0.md)

**Comment by F7085** [comment]
> There is no ResponsiveWrapper in the latest version. See the migration guide:
> 
> https://github.com/Codelessly/ResponsiveFramework/blob/master/migration_0.2.0_to_1.0.0.md
 So, the library is not offering the most important thing, the auto Scale? Now is the same to put a sizedBox with a burned value?

**Comment by F7085** [comment]
> There is no ResponsiveWrapper in the latest version. See the migration guide:
> 
> https://github.com/Codelessly/ResponsiveFramework/blob/master/migration_0.2.0_to_1.0.0.md



adssadasdasdasd

**Comment by abigotado** [comment]
@F7085 No, it provides autoscaling like it used to, but now it is set up differently - using ResponsiveScaleBox and ResponsiveValue instead of ResponsiveWrapper, as it is described in the guide.

**Comment by Eerey** [comment]
I stumbled upon this issue aswell. Please update the readme, because it is indeed confusing.
